### ISSUE TITLE
Some questions about details.

### ISSUE THREAD

**Issue by zxczrx123** [DeyvidKochanov-TomTom/kprnet#4]
Thanks very much for your work!I note that "during training random crops of width 1025 are sampled from the range images" and I'm curious if models trained in this way only and act directly on the raw data will cause a performance loss.
I have trained about 80 epochs with default settings and got 61.8 on the validation set, which differs from the 64.1 in the paper, and I would like to know how to improve the performance further.Can you offer some advice?

**Comment by DeyvidKochanov-TomTom** [comment]
Hi,
 my best guess is the pre-trained weights weren't loaded correctly.
Could you change this line https://github.com/DeyvidKochanov-TomTom/kprnet/blob/master/train_kitti.py#L73 to True and see what happens? The expected outcome is that it will crash but you should see a list of the weights which were not loaded from the checkpoint.
We ran one experiment with the code from the repo before publishing it reaches 63+ mIoU fairly fast. Way faster than 80 epochs In the end there was a small deviation from the result in the paper but it was about 0.2 mIoU so it could be just variance.
![image](https://user-images.githubusercontent.com/64531483/92567752-6f941c00-f27e-11ea-81a0-d5a6df44b1ab.png)

**Comment by zxczrx123** [comment]
Sorry, I overlooked my change to the batch size.For fast training,I use batch size=7 with 8 V100 32G,I think that might be the reason for the difference.Do you think so?I'll also verify if the pre-training loads correctly afterwards.

**Comment by DeyvidKochanov-TomTom** [comment]
I guess doing this okay but then you need to change the learning rate and OHEM threshold

**Comment by zxczrx123** [comment]
Hi! I'm recently trying to build a lightweight but efficient range-view network.What do you think of 2d Unet's performance on range-view?
